recipes/rabbitmq/f52bca4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/sbin/rabbitmq-defaults b/sbin/rabbitmq-defaults
+index b845f74..95dccce 100755
+--- a/sbin/rabbitmq-defaults
++++ b/sbin/rabbitmq-defaults
+@@ -16,7 +16,7 @@
+ ##
+ 
+ ### next line potentially updated in package install steps
+-SYS_PREFIX=${RABBITMQ_HOME}
++SYS_PREFIX=${PREFIX}
```

Comment:
PREFIX is being expanded during install time at recipes/rabbitmq/build.sh:27, is that okay with you?